### DISCUSSION TITLE
hue adore mirror light support (model: 3418231P6)

### DISCUSSION THREAD

**Discussion by moversluizen** [Koenkk/zigbee2mqtt#13326] [Devices]
I would like to get support for the hue adore mirror light so i can use it inside home assistant without needing the hue bridge

**Comment by Koenkk** [comment]
Please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+
